src/xdocs/config_misc.xml

```diff
@@ -166,6 +166,72 @@
       </subsection>
     </section>
 
+    <section name="CommentsIndentation">
+      <subsection name="Description">
+        <p>
+          Controls the indentation inside comments and between comments and surrounding code.
+          There're should be whitespace between comment
+          declaration (//, /*) and it's content. Single-line block comments (/* ... */)
```

Comment:
Please wrap the code wragments with `<tt>` as it is done in e.g. http://checkstyle.sourceforge.net/config_blocks.html#RightCurly